<launch>
  <param name="setpt_update_rate" type="double" value="50"/>
  <param name="velocity_max" type="double" value="200"/>
  <param name="acceleration_max" type="double" value="1000"/>
  <node name="setpt_source" pkg="setpt_source" type="setpt_action_server.py"/>
  <node name="setpt_rel_to_abs" pkg="setpt_source" type="setpt_rel_to_abs.py"/>
</launch>
